static/js/stream_data.js

```diff
@@ -87,6 +87,21 @@ let filter_out_inactives = false;
 const stream_ids_by_name = new FoldDict();
 const default_stream_ids = new Set();
 
+exports.stream_post_policy_values = {
+    everyone: {
+        code: 1,
+        description: i18n.t("Allow everyone to post"),
+    },
+    admins: {
+        code: 2,
+        description: i18n.t("Restrict posting to only organization administrators"),
+    },
+    non_new_members: {
+        code: 3,
+        description: i18n.t("Restrict new members from posting"),
+    },
+};
+
```

Comment:
Hope it was okay that I moved the stream_post_policy_values from stream_edit.js to stream_data.js as I felt that would be a more appropriate place, but it in doing so I had to include `set_global('i18n', global.stub_i18n)` in a way more tests files than if stream_edit was used.

Reply:
Yeah, that's great.